src/cascadia/TerminalApp/Pane.h

```diff
@@ -62,6 +62,16 @@ class Pane : public std::enable_shared_from_this<Pane>
     void ClearActive();
     void SetActive();
 
+    struct BuildStartupState
+    {
+        std::vector<winrt::Microsoft::Terminal::Settings::Model::ActionAndArgs> args;
+        std::shared_ptr<Pane> firstPane;
+        std::optional<uint32_t> focusedPaneId;
+        uint32_t panesCreated;
+    };
+    BuildStartupState BuildStartupActions(uint32_t currentId, uint32_t nextId);

```

Comment:
I initially did that, but was conflicted because I am replicating logic from TerminalTab (which currently sets the ids) into the pane. I prefer not having default arguments (maybe writing rust for so long has conditioned me) but I can be convinced either way.